bigtable-dataflow-parent/bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/batch/common/SplitTracker.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.batch.common;
+
+import com.google.bigtable.repackaged.com.google.protobuf.ByteString;
+
+/**
+ * Determines whether or not a split occurred before a rowKey.
+ *
+ */
+public interface SplitTracker {
+  public boolean isAfterSplit(ByteString rowKey);
+}
```

Comment:
Will we have multiple implementations of this? If not, I'm not sure the abstraction is worth it